GiniVision/Classes/Core/MultipageReviewBottomCollectionCell.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  MultipageReviewBottomCollectionCell.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 2/1/18.
+//
+
+import Foundation
+
+final class MultipageReviewBottomCollectionCell: UICollectionViewCell {
```

Comment:
I really don't understand what a bottom and a top cell is. Isn't there a better name?

Reply:
`MultipageReviewPagesCollectionCell` sound a little bit redundant maybe, but it is more clear in case that we decide to move the collection to other place. 👍